src/styles/product.scss

```
@@ -0,0 +1,10 @@
+@import 'colors';
+
+.card__product {
+  color: $color-text-black;
+  font-size: $font-size;
+  font-style: normal;
+  font-weight: $font-weight;
+  line-height: 18px;
+  margin-bottom: 4;
```

check adding `px` all over the page
```suggestion
  margin-bottom: 4px;
```